### PR TITLE
[OSDOCS-11902]: Adding release note for etcd node scaling

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-11902

Link to docs preview: https://81992--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-etcd-4-5-nodes_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -589,6 +589,11 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-scalability-and-performance_{context}"]
 === Scalability and performance
 
+[id="ocp-4-17-etcd-4-5-nodes_{context}"]
+==== Node scaling for etcd
+
+In general, clusters must have 3 control-plane nodes. However, in this release, if your cluster is installed on a bare metal platform, you can scale a cluster to up to 5 nodes as a post-installation task. The etcd Operator scales accordingly to account for the additional node. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-node-scaling_recommended-etcd-practices[Node scaling for etcd].
```
Comment (kowen-rh) [inline release_notes/ocp-4-17-release-notes.adoc:595]: I'm not sure that we need the introductory sentence here. I'd use something like this:

```adoc
In this release, if your cluster is installed....
```

The rest of the text looks good to me. :+1:

**Review by kowen-rh — COMMENTED** [review]
I'm not sure that the introductory sentence is necessary, but I'll leave that up to you to consider. Otherwise, LGTM! :+1: 

/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (lahinson) [reply release_notes/ocp-4-17-release-notes.adoc:595]: Good point. I'll go with your suggestion. Thanks, Kevin!

**Pull request merged**